src/pages/[platform]/build-a-backend/auth/advanced-workflows/index.mdx

```diff
@@ -375,6 +375,77 @@ func federateToIdentityPoolsUsingCustomIdentityId() async throws {
 }
 ```
 
+## Keychain Sharing
+
+### Migrating to a Shared Keychain
+
+To use a shared keychain:
+
+1. In Xcode, go to Project Settings → Your Target → Signing & Capabilities
+2. Click +Capability
```

Comment:
```suggestion
2. Select +Capability
```

styleguide nit, https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#15-use-carets-to-help-users-navigate-menus-use-select-instead-of-click